consts.go

```diff
@@ -1,32 +1,29 @@
 package wal
 
-const (
-	// pageSize defines the size of a single page in the wal
-	pageSize = 4096
+import "github.com/NebulousLabs/Sia/crypto"
```

Comment:
We are trying not to import any Sia libs into this package, as it's separate. In this case, I think you can import the golang library directly and still have access to all the constants that you need.

The one place I couldn't escape the sia libs was the `build` package to get access to some of its testing helpers.

Reply:
Let's make this a separate PR though, a PR that removes as much of the Sia libs as possible.

The testing libs we have in Sia probably deserve to be split out into their own package anyway.

Reply:
Sounds good. The existing code imported `crypto` to do blake2b hashing. I can just import blake2b directly.